src/_guides/language/sound-problems.md

```diff
@@ -63,10 +63,7 @@ see [Runtime errors](#common-errors-and-warnings).
 error • The <member> '...' isn't defined for the class '...' • undefined_<member>
 ```
 
-These errors usually appear in code where a variable is statically known
```

Comment:
I think we need to add back this text. Maybe make it a bulleted list.

These errors usually appear under one of the following conditions:

* A variable is statically known...
* A generic class has a bounded type parameter, and an instance of the class omits the type argument.

Reply:
Excellent suggestion. Done

Reply:
(Note that I tweak the first sentence. PTAL)